conda_devenv/devenv.py

```diff
@@ -387,11 +385,21 @@ def truncate_history_file(env_directory):
             history.truncate()
 
 
+def _is_mamba_installed() -> bool:
+    try:
+        import mamba
+    except ModuleNotFoundError:
+        return False
+    return True
+
+
 def __call_conda_env_update(args, output_filename):
-    import sys
+    env_manager = "conda"
```

Comment:
Can we instead use `sys.arv[0]` to obtain which program is running? This would make this generic (without knowing about `mamba` at all), and simplify the code a lot.

If that works, we can even discard the `disable_mamba` option.

Reply:
Sure, I am going to change it

Reply:
Sory... I don't think conda_devenv receive the info about the calling program from argv.

Reply:
gonna test it later, but I am not so sure either :grimacing:

Reply:
```
W:\
λ cat \Miniconda\Lib\site-packages\conda_devenv\devenv.py | grep -C5 "============================="


def main(args=None):
    if args is None:
        args = sys.argv[1:]
    print('=============================')
    print(sys.argv)
    return 0
    parser = argparse.ArgumentParser(
        description="Work with multiple conda-environment-like yaml files in dev mode."
    )

W:\
λ conda devenv --version
=============================
['W:\\Miniconda\\Scripts\\conda-devenv', '--version']
```

Reply:
Thanks @prusse-martin !
I will revert the modifications then!
I will do it later, because I am a bit busy right now :grimacing: